Crust/Mapper/MappingOperator.swift

```diff
@@ -290,7 +290,7 @@ private func map<T, U: Mapping, V: Sequence>(
 }
 
 @discardableResult
-public func map<T, U: Mapping, C: MappingContext>(toCollection field: inout U.SequenceKind, using binding:(key: Binding<U>, context: C)) -> C where U.MappedObject == T, U.SequenceKind: RangeReplaceableCollection, U.SequenceKind.Iterator.Element == U.MappedObject, T: Equatable {
+public func map<T, U: Mapping, C: MappingContext, Coll: RangeReplaceableCollection>(toCollection field: inout Coll, using binding:(key: Binding<U>, context: C)) -> C where U.MappedObject == T, Coll.Iterator.Element == U.MappedObject, T: Equatable {
```

Comment:
Might be time for more verbose generic labels? `func map<T, U: Mapping, Context: MappingContext, Collection: RangeReplaceableCollection>(...)`?